### PR TITLE
updating docs with info on Torque, cartoCSS, TorqueCSS, composite operations, data types

### PR DESCRIPTION
**Pull request by michellechandra**
@andrewxhill

### CODE REVIEW

**Comment by andrewxhill** [comment]
Looks great to me.

@matallo would you mind just a quick sanity check for us? it is a big set of changes but it looks good to me locally

File: cartodb-editor.md [hunk 1]
```diff
@@ -175,6 +175,14 @@ You can perform the following administrative actions in the Options menu:
 - **Delete this table**  
   Entirely delete your table. But note that this will also delete any visualizations that rely on the table!
 
+### Supported Data Types
```
Comment (matallo) [inline cartodb-editor.md]: uhm, it makes sense to call these _Supportted data types_ so I think we should replace [Supported data formats](http://docs.cartodb.com/cartodb-editor.html#supported-data-formats) => _Supported file formats_ or consolidate somehow.

File: cartodb-editor.md [hunk 2]
```diff
@@ -175,6 +175,14 @@ You can perform the following administrative actions in the Options menu:
...
+
+When you upload your data to CartoDB, it will automatically assign a data type to your data such as string, number, date or boolean. It's important to check the correct data type was assigned. It is especially important to make sure CartoDB knows if a column is a date type so visualizations that rely on temporal information such as Torque animate correctly.
+
+<p class="wrap-border"><img src="{{ '/img/layout/cartodb-editor/datetype.png' | prepend: site.baseurl }}" alt="Visualizations" /></p>
```
Comment (matallo) [inline cartodb-editor.md]: this looks a bit distorted, a bit too big, is this a retina image or something?

File: cartodb-editor.md [hunk 3]
```diff
@@ -489,11 +554,81 @@ CartoCSS is the syntax language that CartoDB uses to allow users greater control
...
+
+-__Marker Width:__ Change the width of the markers.
+
+#### Composite Operations
+
+Composite operations style the way colors of overlapping markers interact with each other. You can think of them as blend modes or filters, similar to blend operations in Photoshop. Composite operations are available in the Simple, Choropleth, Bubble, Density and Torque visualizations. 
+
+The available options within the wizards interface include multiply, screen, overlay, darken, lighten, color-dodge and color-burn. In addition, blend modes specific to Torque include lighter, multiply, source-over, and xor. For more information on the different options available, refer to the documentation on [Mapbox](https://www.mapbox.com/tilemill/docs/guides/comp-op/).
+
+<p class="wrap-border"><img src="{{ '/img/layout/cartodb-editor/markercompop.png' | prepend: site.baseurl }}" alt="Marker CartoCSS Composite Operations" /></p>
```
Comment (matallo) [inline cartodb-editor.md]: this image also looks huge compared to the rest

**Comment by matallo** [comment]
awesome @brittanymicek! I already made some comments and pushed some style changes

File: cartodb-editor.md (hunk 2 above)
Comment (michellechandra) [inline cartodb-editor.md]: @matallo This is a screenshot I made. I have reduced the image size. Let me know if looks better.

File: cartodb-editor.md (hunk 3 above)
Comment (michellechandra) [inline cartodb-editor.md]: @matallo Reduced the image size on this one.

File: cartodb-editor.md (hunk 1 above)
Comment (michellechandra) [inline cartodb-editor.md]: @matallo I have updated the titles to be Supported File Formats and moved up the Supported Data Types section.

**Comment by matallo** [comment]
looks gorgeous now, merging :sparkles:

**Pull request merged**